enterprise_access/apps/bffs/api.py

```diff
@@ -381,3 +425,17 @@ def transform_enterprise_customer_users_data(data, request, enterprise_customer_
         'staff_enterprise_customer': staff_enterprise_customer,
         'should_update_active_enterprise_customer_user': should_update_active_enterprise_customer_user,
     }
+
+
+def transform_secured_algolia_api_key_response(secured_algolia_api_key_response):
+    """
+    Retrieve the secured algolia api key
+    """
+    catalog_uuids_to_catalog_query_uuids = secured_algolia_api_key_response.get(
+        'catalog_uuids_to_catalog_query_uuids',
+        {}
+    )
+    if algolia := secured_algolia_api_key_response.get('algolia', {}):
+        secured_algolia_api_key = algolia.get('secured_api_key', None)
```

Comment:
nit: `dict.get()` returns `None` by default if the key wasn't found and no default was provided, you don't have to specify it with the second argument here.